content/appointments.md

```diff
@@ -1,49 +1,49 @@
 ---
-description: Free and impartial Pension Wise guidance appointments can help you understand your pension options. Find out how they work, who’s eligible and what you need to do to prepare.
+description: Book a free and impartial Pension Wise guidance appointment to help you understand your pension options.
 ---
 
-#Pension Wise guidance appointments
+You can book if:
 
-You can book a phone or face-to-face guidance appointment to help you understand your pension options. 
+ - you're close to or over 55, and
+ - have a personal or workplace pension based on how much has been paid into your pot - a [defined contribution pension](https://www.pensionwise.gov.uk/pension-types)
 
-You’ll talk about:
+^You can get help with defined benefit pensions at [The Pensions Advisory Service](http://www.pensionsadvisoryservice.org.uk)^
 
-- the different ways you can take money from your pot
-- what each option means for your circumstances
-- next steps you can take
+##How to book
 
-##Speak over the phone or face to face
+Call 0300 330 1001 (from outside the UK: +44 203 733 3495).
```

Comment:
Is this how we want to format the international number?

The norm would be `+44 20 3733 3495`.